x-pack/plugins/security_solution/public/entity_analytics/components/risk_engine_privileges_callout/translations.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCode, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+import { useKibana } from '../../../common/lib/kibana';
+import { CommaSeparatedValues } from '../../../detections/components/callouts/missing_privileges_callout/comma_separated_values';
+import { KIBANA_SAVED_OBJECT_MANAGE_ALL_PRIVILEGE } from '../../../../common/risk_engine';
+import type { MissingPrivileges } from './use_missing_risk_engine_privileges';
+
+export const MISSING_PRIVILEGES_CALLOUT_TITLE = i18n.translate(
+  'xpack.securitySolution.riskEngine.missingPrivilegesCallOut.messageTitle',
+  {
+    defaultMessage: 'Insufficient privileges',
+  }
+);
+
+const kibanaPrivilegesExplanations = {
+  [KIBANA_SAVED_OBJECT_MANAGE_ALL_PRIVILEGE]: i18n.translate(
+    'xpack.securitySolution.riskEngine.missingPrivilegesCallOut.messageBody.kibanaSavedObjectManageAllPrivilegeExplanation',
+    {
+      defaultMessage: '"All" for the "Saved Objects Management" feature under Management',
+    }
+  ),
+} as Record<string, string>;
+
+export const MissingPrivilegesCallOutBody: React.FC<MissingPrivileges> = ({
+  indexPrivileges,
+  clusterPrivileges,
+  kibanaPrivileges,
+}) => {
+  const kibanaPrivilegesExplanation = kibanaPrivileges
+    .map((privilege) => [privilege, kibanaPrivilegesExplanations[privilege]])
+    .filter(Boolean);
```

Comment:
Please consider `reduce` to avoid additional loops

Reply:
Fixed in https://github.com/elastic/kibana/pull/171250/commits/ad34ab5efe1b5fd7b2b35081664ae41c07bc90db

Reply:
Implemented in [ad34ab5](https://github.com/elastic/kibana/pull/171250/commits/ad34ab5efe1b5fd7b2b35081664ae41c07bc90db)